.github/workflows/main.yml

```diff
@@ -23,18 +23,60 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+
+      - name: Prepare environment variables
+        run: |
+          if [ "${{ github.repository }}" != "freeipa/freeipa-operator" ]; then
+            if [ "${{ secrets.ENV }}" == "" ]; then
+              echo "ERROR:You need to define ENV secret with your environment."
+              echo "INFO:See DEVOPS.md file for further information"
+              cat DEVOPS.md
+              exit 1
+            else
+              [ -e "bin" ] || mkdir bin
+              cat > env.ci <<EOF
+          ${{ secrets.ENV }}
+          EOF
+            fi
+          else
+            cp -vf build/ci/env env.ci
+          fi
```

Comment:
Prepare the `env.ci` file; if it is the main repository, it is copied from 'build/ci/env' reopsitory file. For a forked repository, the content of the file with the customs are loaded from the `ENV` secret.